base/settings/tests/testthat/test.addSecrets.R

```diff
@@ -0,0 +1,65 @@
+test_that("`addSecrets` returns settings when `~/.pecan.xml` does not exist", {
```

Comment:
Can you add a few more words of comments explaining what's happening here? Especially: If the user running the tests happens to have a `~/.pecan.xml` after all, will this check work as expected?

Reply:
Thanks for pointing this one out. Added a `stub` to take care of it.